<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Api Check" tests="5" time="0.000">
  <testsuite name="GetPersons" id="2a738944-5ebd-48aa-85b3-1aa982839c7d" timestamp="2022-02-01T08:19:29.979Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: Error: connect ECONNREFUSED 127.0.0.1:5079
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1161:16)

---
]]></system-err>
    <testcase name="Status code is 200" time="0.000" classname="ApiCheck">
      <failure type="AssertionFailure" message="expected { Object (id, _details, ...) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: dc791b5d-0b60-47a8-9b44-66b6a3b13c7a.]]>
        <![CDATA[Collection name: Api Check.]]>
        <![CDATA[Request name: GetPersons.]]>
        <![CDATA[Test description: Status code is 200.]]>
        <![CDATA[Error message: expected { Object (id, _details, ...) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected { Object (id, _details, ...) } to have property 'code'
   at Object.eval sandbox-script.js:1:1).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="PostPerson" id="aff9b163-9ed9-4833-8f02-295e1548d4d0" timestamp="2022-02-01T08:19:29.979Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: Error: connect ECONNREFUSED 127.0.0.1:5079
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1161:16)

---
]]></system-err>
    <testcase name="Status code is 200" time="0.000" classname="ApiCheck">
      <failure type="AssertionFailure" message="expected { Object (id, _details, ...) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: dc791b5d-0b60-47a8-9b44-66b6a3b13c7a.]]>
        <![CDATA[Collection name: Api Check.]]>
        <![CDATA[Request name: PostPerson.]]>
        <![CDATA[Test description: Status code is 200.]]>
        <![CDATA[Error message: expected { Object (id, _details, ...) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected { Object (id, _details, ...) } to have property 'code'
   at Object.eval sandbox-script.js:1:2).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="GetPersonId1" id="37b3da9c-4beb-4c20-95a3-d0c30e6e5f21" timestamp="2022-02-01T08:19:29.979Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: Error: connect ECONNREFUSED 127.0.0.1:5079
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1161:16)

---
]]></system-err>
    <testcase name="Status code is 200" time="0.000" classname="ApiCheck">
      <failure type="AssertionFailure" message="expected { Object (id, _details, ...) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: dc791b5d-0b60-47a8-9b44-66b6a3b13c7a.]]>
        <![CDATA[Collection name: Api Check.]]>
        <![CDATA[Request name: GetPersonId1.]]>
        <![CDATA[Test description: Status code is 200.]]>
        <![CDATA[Error message: expected { Object (id, _details, ...) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected { Object (id, _details, ...) } to have property 'code'
   at Object.eval sandbox-script.js:1:3).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="PutPersonId1" id="59975213-1b52-4563-acdd-22b02c184e6c" timestamp="2022-02-01T08:19:29.979Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: Error: connect ECONNREFUSED 127.0.0.1:5079
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1161:16)

---
]]></system-err>
    <testcase name="Status code is 200" time="0.000" classname="ApiCheck">
      <failure type="AssertionFailure" message="expected { Object (id, _details, ...) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: dc791b5d-0b60-47a8-9b44-66b6a3b13c7a.]]>
        <![CDATA[Collection name: Api Check.]]>
        <![CDATA[Request name: PutPersonId1.]]>
        <![CDATA[Test description: Status code is 200.]]>
        <![CDATA[Error message: expected { Object (id, _details, ...) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected { Object (id, _details, ...) } to have property 'code'
   at Object.eval sandbox-script.js:1:4).]]>
      </failure>
    </testcase>
  </testsuite>
  <testsuite name="DeletePersonId1" id="f5aa7097-a098-4485-b531-7ef4b9f228e1" timestamp="2022-02-01T08:19:29.979Z" tests="1" failures="1" errors="1" time="0.000">
    <system-err><![CDATA[Iteration: 0
RequestError: Error: connect ECONNREFUSED 127.0.0.1:5079
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1161:16)

---
]]></system-err>
    <testcase name="Status code is 200" time="0.000" classname="ApiCheck">
      <failure type="AssertionFailure" message="expected { Object (id, _details, ...) } to have property 'code'">
        <![CDATA[Failed 1 times.]]>
        <![CDATA[Collection JSON ID: dc791b5d-0b60-47a8-9b44-66b6a3b13c7a.]]>
        <![CDATA[Collection name: Api Check.]]>
        <![CDATA[Request name: DeletePersonId1.]]>
        <![CDATA[Test description: Status code is 200.]]>
        <![CDATA[Error message: expected { Object (id, _details, ...) } to have property 'code'.]]>
        <![CDATA[Stacktrace: AssertionError: expected { Object (id, _details, ...) } to have property 'code'
   at Object.eval sandbox-script.js:1:5).]]>
      </failure>
    </testcase>
  </testsuite>
</testsuites>